### DISCUSSION TITLE
404 on all assets after clean install

### DISCUSSION THREAD

**Discussion by tiptap-zz** [directus/directus#6625] [Archive]
Hello,

I've setup a new install of directus, which I am proxy_passing to the port in nginx.

when i view the api domain its correctly jumping to /admin however all of the assets are showing 404.

I'm fairly sure its maybe an nginx error, but can see what

any help appreciated

UPDATE

when i test viewing directus with ip:port the assets loads. so its looking like its an nginx issue

Below is my nginx config

location / {
proxy_pass http://localhost:8055;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
}

Can't work out why nginx wouldnt be showing any of the assets

**Comment by rijkvanzanten** [comment]
Most likely a nginx configuration problem, like you said. I'm no Nginx expert myself, but there seem to be many other threads on the subject that have answers and tips:

https://github.com/directus/directus/discussions?discussions_q=nginx

🙂

**Comment by MoltenCoffee** [comment]
This is my current location block, which works. You'll have to try which one makes the difference (and please let everyone know ;) ):

```nginx
location / {
   proxy_pass http://localhost:2204;
   proxy_http_version 1.1;
   proxy_set_header X-Forwarded-Host $host;   
   proxy_set_header X-Forwarded-Server $host;
   proxy_set_header X-Real-IP $remote_addr;
   proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
   proxy_set_header X-Forwarded-Proto $scheme;
   proxy_set_header Host $http_host;
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection "Upgrade";
   proxy_pass_request_headers on;
   include /etc/nginx/snippets/secure_headers.conf;
}
```

PS: you can add code in Github commens as follows:

~~~
```nginx
 # Code here
```
~~~

**Reply by MoltenCoffee** [reply]
The `include ...` part is of course custom, you can ignore that.
